lib/linguist/languages.yml

```diff
@@ -2507,9 +2507,13 @@ JavaScript:
   interpreters:
   - chakra
   - d8
+  - gjs
   - js
+  - mujs
```

Comment:
I'm having trouble finding examples of this shebang on GitHub. Could the binary be under a different name?

Reply:
No, it's the correct name. Are interpreters subject to the same usage criteria as file-extensions? I was under the impression they weren't (since it's a lot less likely there're two interpreters for supported languages which have the same name).

Reply:
There's still likely a slight overhead (since we do a linear search over interpreters), so I wouldn't expect a threshold as high as for extensions, but there probably should at least be some concrete use cases on GitHub, no?

Reply:
I can't find any examples, so I guess I'll remove them.